app/(marketing)/MarketingHeader.tsx

```
@@ -0,0 +1,90 @@
+          <Image src="/logo-white.svg" alt="Helper" width={120} height={32} />
+        </Link>
+        <div className="flex items-center gap-2 md:gap-4">
+          <Link
+            href="https://github.com/antiwork/helper"
+            target="_blank"
+            rel="noopener noreferrer"
+            className="flex items-center group"
+          >
+            <span className="inline-flex items-center justify-center w-9 h-9">
+              <span
+                dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)